nipype/interfaces/ants/registration.py

```diff
@@ -49,8 +49,12 @@ class ANTSInputSpec(ANTSCommandInputSpec):
     metric = traits.List(traits.Enum('CC', 'MI', 'SMI', 'PR', 'SSD',
                                      'MSQ', 'PSE'), mandatory=True, desc='')
 
-    metric_weight = traits.List(traits.Float(), requires=['metric'], desc='')
-    radius = traits.List(traits.Int(), requires=['metric'], desc='')
+    metric_weight = traits.List(traits.Float(), value=[1.0], usedefault=True,
+                                requires=['metric'], mandatory=True,
+                                desc='the metric weight(s) for each stage. '
+                                'The weights must sum to 1 per stage.')
+
+    radius = traits.List(traits.Int(), requires=['metric'],  mandatory=True, desc='')
```

Comment:
no description added.

Reply:
I added some description, but originally none of the traits had any description, and most of the still don't have. I can write the description when working on tests.  I guess we can keep this PR open for some time

There is any typical value that could be used as default?